examples/07-technology-showcase-examples/20-example-technology-showcase-dynamic-simulation-PCB.py

```diff
@@ -213,48 +218,55 @@
 ###############################################################################
 # Post-process PSD analysis
 # ~~~~~~~~~~~~~~~~~~~~~~~~~
-# Using MAPDL methods in POST1
+# The response spectrum analysis is post-processed. First, the standard
+# MAPDL POST1 postprocessor is used. Then, the MAPDL time-history
+# POST26 postprocessor is used to generate the response power spectral
+# # density. Note - the graph generated through POST26 is exported as a picture
+# in the working directory. Finally, the results from POST26 are saved to Python
+# variables to be plotted in the Python environment with the use of Matplotlib library.
```

Comment:
```suggestion
# density. 
# .. note:: The graph generated through POST26 is exported as a picture
#    in the working directory. Finally, the results from POST26 are saved to Python
#    variables to be plotted in the Python environment with the use of Matplotlib library.
```

Reply:
This will add a "note" like this:

![image](https://user-images.githubusercontent.com/28149841/157847952-7bda42c7-0d70-4726-82eb-47eda552ed2c.png)

Reply:
Oh, awesome! I'll take it!